### ISSUE TITLE
Problems with integrity check after activating social_login

### ISSUE THREAD

**Issue by rollanders** [zorn-v/nextcloud-social-login#149]
After activating and configuring social_login I have a warnig about Integrity check failed.  
The Social Login works OK. 

Ubuntu18.0.4, NC17.0.3, Nginx,  

 Technical information
=====================
The following list covers which files have failed the integrity check. Please read
the previous linked documentation to learn more about the errors and how to fix
them.

Results
=======
- core
	- EXTRA_FILE
		- .well-known/index.html
		- .well-known/microsoft-identity-association.json
		- googleccf391ca0acdee1b.html

Raw output
==========
Array
(
    [core] => Array
        (
            [EXTRA_FILE] => Array
                (
                    [.well-known/index.html] => Array
                        (
                            [expected] => 
                            [current] => a3c177970334b753a103980fe40747cf092fa70285f2c45375aed0e11b231bee7970e26e43d93f98c4aede8ad9e404460aaed1551e80f16ed30c743754c9ac6c
                        )

                    [.well-known/microsoft-identity-association.json] => Array
                        (
                            [expected] => 
                            [current] => 73d796b34d7a874949bfbcbc6b5c98ec9c72e19e0f4ed7798c8e599a48c341b0d11df297ed952778a114df97a7a8e3e34f526e5902d7612d35526bb1efc1b632
                        )

                    [googleccf391ca0acdee1b.html] => Array
                        (
                            [expected] => 
                            [current] => c55dbf334b4a9386a14108bf6dc1b7887c0fee237f15a9e863651f2d183367a209670917e45a2fb73fc40902ead10d68b0aae1ffdef634125a3110fb3cf08a43
                        )

                )

        )

)

**Comment by zorn-v** [comment]
Not related to social login. It is your server and certbot configuration possibly

**Comment by rollanders** [comment]
Hm. I do not understand. 

The google and microsoft-identity-association files must surely be associated with social login. I suspect that they are involved in the authentication through google or microsoft.
 
I have no other app that has any relation to google or MS. These errors came after I activated the social login.  But it may have come after upgrading to NC17.0.3 which I did almost concurrently. 

I guess I should deactivate, reinstall and reconfigure social-login.
